sources/packages/backend/apps/db-migrations/src/sql/Queue/Add-queue-configurations-values.sql

```diff
@@ -0,0 +1,21 @@
+-- Inserting values to 'queue_configurations' table. This will be part of table creation migration. So, no drop migration sql, this data will destroy with table in drop migration.
+INSERT INTO
+  queue_configurations(name, configuration)
+VALUES
+  (
+    'start-application-assessment',
+    '{ "attempts": 3, "backoff": 180000, "dashboardReadonly": false }' :: json
+  );
+
+INSERT INTO
```

Comment:
I was under the impression that we would try to add all configurations in this PR to avoid creating individual migrations.
Is it not part of this task?